.travis.yml

```diff
@@ -16,6 +25,18 @@ script:
     -e token
     my-test-image ./todo_app/test/test_trello_app.py
 
+after-script:
+  - docker push $DOCKER_USERNAME/todo-app:$TRAVIS_COMMIT
+
+before_deploy:
+  - docker push registry.heroku.com/$HEROKU_APP/web
+
+deploy:
+  provider: script
+  script: heroku container:release web --app $HEROKU_APP
+  on:
+    branch: Module-8
```

Comment:
Don't forget to update this to master on merge

Reply:
Yes sure after the pull request is approved I will update this to master on merge